src/lib/cmpManager.js

```diff
@@ -61,6 +61,7 @@ export default class CmpManager {
 	 */
 	notify = (event, data) => {
 		log.info(`Notify event: ${event}`);
+		this.eventData[event] = data;
```

Comment:
Wouldn't be better to extend this object instead of overriding it? I know it won't be useful right away, but it would allow us to store such structure and it could be little bit safer (not loosing any property): 
`
{
    'success': true,
    'smth': 'smth',
    'section': 'details'
}
`

Reply:
The question is whether we really want to merge parameters? I assumed that we want to restore last state. So it was intentional to override whole data.